lib/cpp/test/OneWayHTTPTest.cpp

```diff
@@ -202,10 +202,10 @@ BOOST_AUTO_TEST_CASE( JSON_BufferedHTTP )
 #endif
 
   {
-    std::shared_ptr<TSocket> socket(new TSocket("localhost", port));
+    std::shared_ptr<TSocket> socket(new TSocket("127.0.0.1", port));
```

Comment:
This appears to be an issue with `localhost` getting resolved to the IPv6 address `::1` and stuff breaks.
I am actually reconsidering this commit and instead for the pipeline purge the IPv6 `::1     localhost` from the `/etc/hosts` file in the pipeline.